ui/tests/acceptance/stepDefinitions/helloContext.js

```diff
@@ -0,0 +1,8 @@
+const {client} = require("nightwatch-api")
+
+const {Given} = require("cucumber")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after '{'.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)